sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -81,15 +85,26 @@ def create_properties(user_agent=None):
     return properties
 
 
-def renewable_start_time(renewable):
+def get_renewable_start_time(renewable):
     try:
         return renewable._received_timestamp_utc  # pylint: disable=protected-access
     except AttributeError:
         pass
     try:
         return renewable._session_start  # pylint: disable=protected-access
     except AttributeError:
-        raise TypeError("Registered object is not renewable.")
+        raise TypeError("Registered object is not renewable, renewable must be" +
+                        "a ReceivedMessage or a sessionful ServiceBusReceiver")
+
+
+def get_renewable_lock_duration(renewable):
+    # type: (Union[ServiceBusReceivedMessage, BaseSession]) -> datetime.timedelta
+    # pylint: disable=protected-access
+    try:
+        return max(renewable.locked_until_utc - utc_now(), datetime.timedelta(seconds=0))
+    except AttributeError:
+        raise TypeError("Registered object is not renewable, renewable must be" +
+                        "a ReceivedMessage or a sessionful ServiceBusReceiver")
```

Comment:
same comment as above:
```suggestion
                        "a ServiceBusReceivedMessage or a ServiceBusSession from a sessionful ServiceBusReceiver.")
```